ddpui/services/org_cleanup_service.py

```
@@ -0,0 +1,358 @@
+
+        for org_task in OrgTask.objects.filter(org=self.org, task__type__in=["airbyte"]).all():
+            logger.info(f"will delete connection {org_task.connection_id} in Airbyte and DB")
+            if org_task.connection_id:
+                try:
+                    if not self.dry_run:
+                        airbyte_service.delete_connection(
+                            self.org.airbyte_workspace_id, org_task.connection_id
+                        )
+                        logger.info(f"deleted connection in Airbyte - {org_task.connection_id}")
+                except Exception:
+                    pass
```

Ruff (0.14.10)

`try`-`except`-`pass` detected, consider logging the exception

(S110)